api/app/src/main/kotlin/packit/service/RunnerService.kt

```diff
@@ -66,4 +69,21 @@ class BaseRunnerService(
         runInfoRepository.save(runInfo)
         return res.taskId
     }
+
+    override fun getTaskStatus(taskId: String): RunInfoDto
+    {
+        val taskStatus = orderlyRunnerClient.getTaskStatuses(listOf(taskId), true)[0]
+
+        val runInfo = runInfoRepository.findByTaskId(taskId)!!
```

Comment:
We can be sure there will be a task info (for a valid request) in there because we saved it on submit?

Reply:
yep exactly

Reply:
its probs best to do just do a null check here... just for the times people call the get status endpoint before a submit.. can throw error then . thus can probs check the DB first before we send the request off to orderly.runner.

Reply:
done